WIP fleet management documentation (#705)
neo4j/docs-aura: modules/ROOT/pages/fleet-management/deployment-details.adoc

[[sm-deployment-details]]
= Deployment details
:description: This page describes deployment details

From the deployment card or deployment table, select the more menu *(…​)*.

> In the graph view, the same menu can be accessed by right-clicking the DBMS node. Then select *Show details* from the menu options.

Reviewer: ```suggestion
To access deployment details, use the more menu *(…​)* from the deployment card or deployment table.
In the graph view, the same menu can be accessed by right-clicking the DBMS node. 
```

Reviewer 2: I take it which item to select is considered obvious?

On the right side of the header, the view selector allows you to choose between two views: *table* view or *graph* view (see xref:fleet-management/graph-view.adoc[Graph view]).

== Table view

> The table view shows topology data in a tabular format. Three different tables are selectable using the dropdown in the table header.

Reviewer: ```suggestion
The table view shows topology data in a tabular format. 
Three different tables are selectable using the dropdown in the table header.
```

* Servers
* Databases

> * Database instances

Reviewer: This is problematic, as we have discussed. But not only from a terminology standpoint, it also assumes that the local deployment is a cluster. Can that be safely assumed?

Reviewer 2: Even single-node "clusters" will have an entry for each database under database instances, and this is the only place to access some metadata.

Reviewer: Hmm, I wonder is it is possible to put in a little note or something to explain?

Author: Nothing we can do before GA.

=== Servers

The *Servers* table shows all servers in the deployment.

> For more information, see the link:https://neo4j.com/docs/operations-manual/current/clustering/monitoring/show-servers-monitoring/[documentation] for instructions on how to install plugins. for the `SHOW SERVERS` procedure, which is called by the plugin to obtain the data.

Reviewer: ```suggestion
For more information, see the link:https://neo4j.com/docs/operations-manual/current/clustering/monitoring/show-servers-monitoring/[Operations Manual -> Monitor servers] for instructions on how to install plugins. for the `SHOW SERVERS` procedure, which is called by the plugin to obtain the data.
```

Reviewer: And yeah, that last sentence 🤷‍♀️

=== Databases and Database instances

The *Databases* table shows all databases in the deployment.

> A *Database instance* in this context refers to an instance of a logical database running on a specific server.

Reviewer: This would be a logical spot to explain that a database instance refers to an instance of a database in a cluster.

> For more information, see the link:https://neo4j.com/docs/operations-manual/current/database-administration/standard-databases/listing-databases/[documentation] for instructions on how to install plugins. for the `SHOW DATABASES` procedure, which is called by the plugin to obtain the data.

Author: I guess this is unfinished?

Reviewer 2: There is clearly a copy-paste error with that link! I did not really want to document every single field and what it means, and where we get it from. So I thought I could be lazy and just link to the SHOW DATABASES documentation. I'm open to suggestions here, or feel free to push something to the PR.

> For more information, see the link:https://neo4j.com/docs/operations-manual/current/database-administration/standard-databases/listing-databases/[documentation] for instructions on how to install plugins. for the `SHOW DATABASES` procedure, which is called by the plugin to obtain the data.

Reviewer: ```suggestion
For more information, see the link:https://neo4j.com/docs/operations-manual/current/database-administration/standard-databases/listing-databases/[Operations Manual -> List databases] for instructions on how to install plugins.
for the `SHOW DATABASES` procedure, which is called by the plugin to obtain the data.
```
The last sentence I'm not sure about?
